src/plugins/auto/plugin.cpp

```diff
@@ -416,6 +416,12 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         }
         // reset the strDevices to support devices
         strDevices = "";
+        auto devicesWithPriority = GetValidDevice(supportDevices, networkPrecision);
```

Comment:
let's add some comment to code, indicating why we have to call GetValidDevice here.

Reply:
Done.